doc/source/conf.py

```diff
@@ -240,7 +240,7 @@
 
 html_theme_options = {
     "external_links": [],
-    "footer_items": ["pandas_footer", "sphinx-version"],
+    "footer_start": ["pandas_footer", "sphinx-version"],
     "github_url": "https://github.com/pandas-dev/pandas",
     "twitter_url": "https://twitter.com/pandas_dev",
     "google_analytics_id": "UA-27880019-2",
```

Comment:
For the error in the CI seems like this config has veen removed: `WARNING: unsupported theme option 'google_analytics_id' given`

Can you check on the pydata theme docs which is the new way to do this please?